observability/metrics/csi-linode-controller-metrics-service.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
```

Comment:
can this be moved into the helm chart templates and conditionally created if the related metrics endpoints are enabled?